packages/Select/src/select-value.scss

```diff
@@ -0,0 +1,51 @@
+@use '~@igloo-ui/tokens/dist/base10/variables' as tokens;
+
+:root {
+  /* Default */
+  --ids-dropdown-value-padding: #{tokens.$space-2} #{tokens.$space-3};
+  --ids-dropdown-value-gap: #{tokens.$space-1};
+
+  /* Compact **/
+  --ids-dropdown-value-padding-compact: #{tokens.$space-1} #{tokens.$space-3};
+
+  /* Disabled */
+  --ids-dropdown-value-color-disabled: #{tokens.$grey-500};
+
+  /* Placeholder */
+  --ids-dropdown-value-color-placeholder: #{tokens.$grey-400};
+}
+
+.ids-select-value {
+  display: flex;
+  flex-flow: row nowrap;
+  align-content: center;
+  gap: var(--ids-dropdown-value-gap);
```

Comment:
is not supported in IE11 :(

Reply:
IE11 is dead since mid-june, did we still support it?

Reply:
Ok, I will change that 👍